sdk/src/azure/platform/az_pahoasync5.c

```diff
@@ -429,6 +429,7 @@ AZ_NODISCARD az_result az_mqtt5_property_bag_clear(az_mqtt5_property_bag* proper
 {
   _az_PRECONDITION_NOT_NULL(property_bag);
 
+  MQTTProperties_free(property_bag->pahoasync_properties);
   property_bag->pahoasync_properties->count = 0;
```

Comment:
I think this line is now redundant - is this done within MQTTProperties_free?

Reply:
It does zero out all fields, removed line.